pokedex.json

```diff
@@ -45006,7 +45006,7 @@
     "description": "It evolved after experiencing numerous fights. While crossing its arms, it lets out a shout that would make any opponent flinch.",
     "evolution": {
       "prev": [
-        "linoone",
+        "264",
         null
```

Comment:
There are still a few nulls left in the json for example here

Reply:
I just removed all of the nulls and replaced them with the values they should be.